src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useHistory, useLocation, useRouteMatch } from 'react-router-dom';
+import { Add } from '@mui/icons-material';
+import { routes } from '../../utils/routes';
+import PageTitle from '../../layouts/PageTitle/PageTitle';
+import { Tab, Tabs } from '@mui/material';
+import { useAuth } from '../../hooks/auth.hooks';
+import { useAllChangeRequests } from '../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { useState, useMemo, useEffect } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { NERButton } from '../../components/NERButton';
+import { isGuest, isLeadership, isHead } from 'shared';
+import PageBlock from '../../layouts/PageBlock';
+
+const ChangeRequestsOverview: React.FC = () => {
+  const history = useHistory();
+  const { isLoading, isError, error } = useAllChangeRequests();
+
+  const auth = useAuth();
+
+  // Values that go in the URL depending on the tab value
+  const viewUrlValues = useMemo(() => ['overview', 'all-change-requests'], []);
+
+  const match = useRouteMatch<{ tabValueString: string }>(`${routes.CHANGE_REQUESTS}/:tabValueString`);
+  const tabValueString = match?.params?.tabValueString;
+
+  // Default to the "overview" tab
+  const initialValue: number = viewUrlValues.indexOf(tabValueString ?? 'overview');
+  const [value, setValue] = useState<number>(initialValue);
```

Comment:
Can we give more details on this useState variable? For instance, we can define this useState variable as [tabIndex, setTabIndex] so the readers know what this variable means.